.devcontainer/devcontainer.json

```diff
@@ -2,7 +2,6 @@
 // README at: https://github.com/devcontainers/templates/tree/main/src/python
 {
   "name": "Pycytominer Dev",
-  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
```

Comment:
Minor note here: is this comment no longer true? It looks like a useful note to retain from a developer understanding / education perspective. I can also understand why it was removed if we're focusing on making this file an opinionated implementation.

Reply:
This is a leftover from the devcontainer template. I'll remove.